tests/test_ebs.py

```diff
@@ -432,6 +433,22 @@ def test_matched(self):
         assert perms == [{"UserId": "112233445566"}]
 
 
+class SnapshotVolumeFilter(BaseTest):
+
+    def test_ebs_volume_filter(self):
+        factory = self.replay_flight_data("test_ebs_volume_related_filter")
+        p = self.load_policy(yaml_load("""
```

Comment:
we generally want to avoid yaml_load in tests, its slow as a pattern compared to using a dictionary, especially in aggregate across many tests.

Reply:
sounds good